src/Model/CollectionInfoIterator.php

```diff
@@ -26,6 +26,7 @@
  * This iterator is used for enumerating collections in a database.
  *
  * @see \MongoDB\Database::listCollections()
+ * @template-extends Iterator<int, CollectionInfo>
```

Comment:
Related to my earlier question, I assume there's no notation for a list-like Traversable?